website/docs/automation/gitops/argocd/update.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Updating the application"
+sidebar_position: 20
+---
+
+During the development lifecycle, there are new container images or the development team may want to change the configuration of the application. You need to update the application to reflect the latest changes following the GitOps workflow.
+
+In this lab excerise, we'll update the `catalog` application to increase replicas to handle higher traffic with Argo CD. 
+
+Update replicas of the `catalog` application in the `kuztomization` file to 3:
+
+```file
+automation/gitops/argocd/update-app/kustomization.yaml
+```
+
+:::tip In your environment
+You should update the same `kustomization.yaml` file and commit to the Git repository.
```

Comment:
I would rephrase with something like:

Normally, you should update your files and commit in the Git repository. In this lab exercice, we provide you with a new version of the file to simulate that we have made a change, and we will instead update the path for our argocd-demo application with the new file.